src/types/grpc.d.ts

```diff
@@ -180,61 +186,73 @@ export namespace credentials {
    * Create an SSL Credentials object. If using a client-side certificate, both
    * the second and third arguments must be passed.
    */
-  export function createSsl(rootCerts: Buffer, privateKey?: Buffer, certChain?: Buffer): CallCredentials;
+  export function createSsl(
+    rootCerts: Buffer,
+    privateKey?: Buffer,
+    certChain?: Buffer,
+  ): CallCredentials;
 
   /**
    * Combine any number of CallCredentials into a single CallCredentials object.
    */
-  export function combineCallCredentials(...credentials: CallCredentials[]): CallCredentials;
+  export function combineCallCredentials(
+    ...credentials: CallCredentials[]
+  ): CallCredentials;
 
   /**
    * Combine a ChannelCredentials with any number of CallCredentials into a
    * single ChannelCredentials object.
    */
-  export function combineChannelCredentials(channelCredential: ChannelCredentials,
-    ...callCredentials: CallCredentials[]): ChannelCredentials;
+  export function combineChannelCredentials(
+    channelCredential: ChannelCredentials,
+    ...callCredentials: CallCredentials[]
+  ): ChannelCredentials;
 
   /**
    * Create a gRPC credential from a Google credential object.
    * todo(connor4312): type
    */
-  export function createFromGoogleCredential(googleCredential: any): CallCredentials;
+  export function createFromGoogleCredential(
+    googleCredential: any,
+  ): CallCredentials;
 
   /**
    * IMetadataGenerator can be passed into createFromMetadataGenerator.
    */
   export interface IMetadataGenerator {
-    (
-      target: { service_url: string },
-      callback: (error: Error | null, metadata?: Metadata) => void,
-    ): void;
+    (target: { service_url: string }, callback: (
+      error: Error | null,
+      metadata?: Metadata,
+    ) => void): void;
   }
 
   /**
    * Create a gRPC credential from a Google credential object.
    * todo(connor4312): type
    */
-  export function createFromMetadataGenerator(generator: IMetadataGenerator): CallCredentials;
+  export function createFromMetadataGenerator(
+    generator: IMetadataGenerator,
+  ): CallCredentials;
 }
 
 export const status: Readonly<{
-  OK: 0,
-  CANCELLED: 1,
-  UNKNOWN: 2,
-  INVALID_ARGUMENT: 3,
-  DEADLINE_EXCEEDED: 4,
-  NOT_FOUND: 5,
-  ALREADY_EXISTS: 6,
-  PERMISSION_DENIED: 7,
-  RESOURCE_EXHAUSTED: 8,
-  FAILED_PRECONDITION: 9,
-  ABORTED: 10,
-  OUT_OF_RANGE: 11,
-  UNIMPLEMENTED: 12,
-  INTERNAL: 13,
-  UNAVAILABLE: 14,
-  DATA_LOSS: 15,
-  UNAUTHENTICATED: 16
+  OK: 0;
```

Comment:
This is an object map of strings to numbers, not a true TypeScript enum (where numbers map to the enum key names and key names map to numbers).

Caps is not pretty, but it's what GRPC has